### ISSUE TITLE
xcode 9.2.0 is no longer supported by CircleCI

### ISSUE THREAD

**Issue by mauzey1** [PCMDI/cmor#555]
Recent CircleCI macOS tests have failed with the following error:
```
Build-agent version 1.0.18336-07e4d1b1 (2019-10-28T09:24:47+0000)
Creating a dedicated VM with xcode:9.2.0 image
failed to create host: Image xcode:9.2.0 is not supported

failed to create host: Image xcode:9.2.0 is not supported
```

I plan to update the tests to use xcode 11.1.0.

**Comment by mauzey1** [comment]
I was having problems using xcode 11.1.0 but xcode 10.1.0 seems to work fine so I will upgrade to that version instead.

**Comment by mauzey1** [comment]
I was able to fix the issues I had with xcode 11.1.0 by removing changes to the CFLAGS when building for OSX.
